roles/ceph-common/defaults/main.yml

```diff
@@ -135,6 +135,7 @@ ceph_stable_ice_kmod: 3.10-0.1.20140702gitdc9ac62.el7.x86_64
 # on RHEL 7.
 #
 ceph_stable_rh_storage: false
+ceph_stable_rh_storage_version: 1.3 # next version is 2
```

Comment:
The problem here is that "1.3" will cover "1.3.1" as well as "1.3.2" because it is the same repo. Adding a link that shows all versions available (e.g. https://access.redhat.com/documentation/en/red-hat-ceph-storage/ ) will make this break if someone goes strictly by released version.